IdokladSdk/Authentication/Extensions/RestClientExtensions.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using System.Net;
+using System.Runtime.CompilerServices;
 using System.Security.Authentication;
 using IdokladSdk.Authentication.Models;
 using IdokladSdk.Exceptions;
 using Newtonsoft.Json;
 using RestSharp;
 
+[assembly: InternalsVisibleTo("IdokladSdk.UnitTests")]
```

Comment:
Asi detail, ale pokud se ten atribut týká všech internal metod v celém assembly, viděl bych ho radši v nějakém "globálním" zdrojáku, ne zde.

Reply:
momentálne sa testuje len táto jedna trieda, až sa rozhodneme otestovať aj ďalšie, tak by som to zmenil až vtedy, zbytočne by som to celé neotváral, i keď je to len pre testy